app/controllers/onboarding_controller.rb

```diff
@@ -17,28 +17,42 @@ def create
     return redirect_to_success if User.email_taken?(user_params[:email])
```

Comment:
We need to invalidate the token in this case as well. Otherwise the following scenario would be possible:

* User A has already registered for 100eyes using email.
* User B hasn’t yet registered and has a valid invite link and knows the email address of user A.
* User B enters user A’s email address into the onboarding form.
* User B is redirected to the success page, but the token is still valid.
* User B knows that user A is a 100eyes contributor.

This is a potential privacy flaw.

Reply:
yeah, that's a good point